src/CSS/button.css

```diff
@@ -0,0 +1,63 @@
+.card-btn-link{
+    width:100%;
+}
+
+.btn-primary{
+    background-color: var(--primary-dark-color) !important;
+    font-size: 1.5rem !important;
+    min-width: 8rem;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color) !important;
+    font-size: 1.5rem !important;
+}
+
+.btn-outline-primary{
+    border: 2px solid var(--primary-dark-color) !important;
```

Comment:
you should work on its specificity. instead use of important.

Reply:
I use this to overwrite component library defined property